plugins/coverbrowser.koplugin/xutil.lua

```diff
@@ -13,7 +13,7 @@ int compress2(uint8_t *dest, unsigned long *destLen,
 int uncompress(uint8_t *dest, unsigned long *destLen,
                const uint8_t *source, unsigned long sourceLen);
 ]]
-local zlib = ffi.load(ffi.os == "Windows" and "zlib1" or "z")
+local zlib = ffi.load(ffi.os == "Windows" and "zlib1" or "libs/libz.so.1")
```

Comment:
z is short for libz(.so/.dynlib/.dll/etc) and should be kept.

What is the precise error message? If it came from here then that's where it should be fixed:

https://github.com/koreader/koreader/blob/e21fe5504299f2c650939da22adefb73aa4f0f2b/setupkoenv.lua#L19-L31

Also pinging @poire-z who wrote the zlib stuff here.

Reply:
> z is short for libz(.so/.dynlib/.dll/etc) and should be kept.

By this I mean that at the very least what you wrote should be `or "z" *or* "new_addition"` even if it'd still be the wrong place.